### PR TITLE
chore(design): translate tracepoints definition

### PR DESCRIPTION
**Pull request by takam5f2**

### CODE REVIEW

File: docs/design/tracepoint_definition.md [hunk 1]
```diff
@@ -1,30 +1,26 @@
...
->
-> CARET は基本的には実装と切り離すために、LD_PRELOAD を利用したフックによるトレースポイントの追加をしています。  
-> ただし、rclcpp には、プロセス内通信などのテンプレートによる実装を含み、LD_PRELOAD ではフックできない処理があります。  
-> LD_PRELOAD は実行時の動的リンクに対しフックできる機能であるのに対し、  
-> テンプレートで実装された処理はコンパイルでバイナリ内に組み込まれてしまうためフックできません。  
-> このようなテンプレートで実装された処理に対してのみ、rclcpp をフォークしてトレースポイントを追加しています。  
-> このような理由から、測定対象のアプリケーションを再ビルドする必要が生じています。
+<prettier-ignore-start>
+!!! info
+    Please read this section if you are interested in CARET-dedicated tracepoints are extended by the forked rclcpp and LD_PRELOAD. CARET would like to add tracepoints by function hooking as possible. LD_PRELOAD is reasonable to hook functions defined in dynamic library, but it cannot be applied to functions by implemented with C++ template. Such template-based implementation is mapped into binary file after it is built or compiled. Original rclpp uses C++ template for some functions like intra-process communication, for example. The forked rclcpp is introduced to add tracepoints to the functions.
```
Comment (hsgwa) [inline docs/design/tracepoint_definition.md:20]: Original rclpp
Original rcl**c**pp

File: docs/design/.pages [hunk 2]
```diff
@@ -1,5 +1,5 @@
 nav:
-    - ソフトウェアアーキテクチャ: design.md
-    - トレースポイントの定義: tracepoint_definition.md
+    - Architecture overview: architecture_overview.md
+    - Supported tracepoints: supported_tracepoint.md
```
Comment (hsgwa) [inline docs/design/.pages:3]: supported_tracepoint**s**.md

File: docs/design/supported_tracepoints.md [hunk 3]
```diff
@@ -652,11 +647,10 @@ graph LR
...
 
-callback_start から message_addr/source_timestamp までが一意に紐付けられる。  
-また、publisher 側のグラフで示した通り、message_addr/source_timestamp から publish までが一意に紐付けられる。  
-したがって、publish から callback_start までは一意に紐付けることができる。
+`message_addr` and `source_timestamp` is mapped a corresponding `callback_start`.  
+As explained, `message_addr` and `source_timestamp` are identifier for transmission flow using `publish` method.
+This means that a invoked `publish` method is mapped to a `callback_start`.
 
-ただし、コールバックの実行と publish の紐付けはできていないので注意。
+However this does not mean that CARET can map a `callback_start` to a corresponding `publish`. We can trace a certain flow from `publish` to `callback_start`, but the reversed flow is not out of CARET's capability.
```
Comment (hsgwa) [inline docs/design/supported_tracepoints.md:656]: Could yout add following sentences?

> The mapping between callback_start and publish will be improved in v0.3.* release or later.

**Review by hsgwa — CHANGES_REQUESTED** [review]

**Comment by takam5f2** [comment]
@hsgwa  
I fixed sentences as you commented. Please check it.
Best,

File: docs/design/supported_tracepoints.md (hunk 3 above)
Comment (takam5f2) [reply docs/design/supported_tracepoints.md]: Fixed in https://github.com/tier4/CARET_doc/pull/40/commits/e70eda1881e8fc8214f745c3bda1aeea170ae96b

File: docs/design/.pages (hunk 2 above)
Comment (takam5f2) [reply docs/design/.pages]: Fixed in https://github.com/tier4/CARET_doc/pull/40/commits/e70eda1881e8fc8214f745c3bda1aeea170ae96b

File: docs/design/tracepoint_definition.md (hunk 1 above)
Comment (hsgwa) [reply docs/design/tracepoint_definition.md]: @takam5f2 Could you fix this typo?

Comment (takam5f2) [reply docs/design/tracepoint_definition.md]: fixed in https://github.com/tier4/CARET_doc/pull/40/commits/f15fff3805893c68a233edcbfa7d330026cf0265

**Review by hsgwa — APPROVED** [review]
LGTM

**Pull request merged**